Resources/views/base.html.twig

```diff
@@ -56,76 +59,57 @@
 </div>
 <![endif]-->
 
-<div id="main-wrapper" class="main-wrapper">
+<div id="main-wrapper" class="main-wrapper" data-sidebar="wrapper">
```

Comment:
Het is niet perse nodig een data attribuut een value te geven als je dat niet echt nodig hebt, als je dat niet uitleest. Hier zou ik dus `data-sidebar-wrapper`, ik probeer daar sinds kort ook op te letten en vind dat ook leesbaarder en properder